Source/Events/OnAuctionHouseClosed.lua

```diff
@@ -3,13 +3,19 @@ function Auctionator.Events.OnAuctionHouseClosed()
 
   AuctionatorAHFrame:Hide()
 
-  if Auctionator.FullScan.State.InProgress and not Auctionator.FullScan.State.Completed then
+  if Auctionator.FullScan.State.InProgress then
     Auctionator.FullScan.State.InProgress = false
-
-    Auctionator.Utilities.Message(
-      "Full scan failed to complete. " ..
-      Auctionator.FullScan.NextScanMessage()
-    )
+    if not Auctionator.FullScan.State.QuickCompleted then
```

Comment:
What is the distinction between `QuickCompleted` and `DetailedCompleted`?

Reply:
The scan runs twice, once with the limited information available immediately (20s, tops), but the item links (and names for pets) are often not available immediately and require waiting for further events. I've tried a few things, including waiting for all the events to fire, but the link/name only seems to be available with the event that fires to return it